docusaurus/docs/how-to-guides/data-source-plugins/data-source-configuration.md

```diff
@@ -0,0 +1,135 @@
+---
+id: data-source-configuration
```

Comment:
is this a duplicated version of the previous file?

Reply:
Yes, sorry about that. The files should basically be identical, but I didn't intend the one above to be checked in; it should be deleted. I'll clean this up after we get David's response to your questions since your comments are on the above file.

Reply:
removed